drift_demo.yaml

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure Security Groups do not have unrestricted specific ports open - (SSH,22)**
 
**Vulnerability**
Security Groups - Unrestricted Specific Ports - SSH,22. Exposed ports may allow attackers to access the application listening on that port.
 
**Remediation**
In AWS Console - 
 1. Sign in the AWS Console and go to the Amazon VPC console.
 2. In the navigation pane, select Security Groups.
 3. For each security group, perform the following actions:
      a) Select the security group.
      b) Select the Inbound Rules.
      c) Identify the rules to be removed.
      d) Remove the required rules.
 4. Click Save.

source: tenablecs-